commcare_connect/templates/opportunity/user_tasks.html

```
@@ -22,7 +22,22 @@
     <div class="relative">
       {% include 'components/worker_page/visit_task_tabs.html' %}
-      <div class="absolute right-4 top-1/2 -translate-y-1/2">
+      <div class="absolute right-4 top-1/2 -translate-y-1/2 flex items-center gap-2">
+        {% if can_manage_tasks %}
+          <button type="button"
+                  class="button-icon shadow-sm"
+                  @click.stop="showCreateTaskModal = true"
+                  x-tooltip.raw="{% trans 'Create Task' %}">
+            <i class="fa-solid fa-plus text-brand-deep-purple"></i>
+          </button>
+          <button type="button"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)